office-addin/package.json

```diff
@@ -12,7 +12,7 @@
     "dev_server_port": 3000
   },
   "scripts": {
-    "build": "webpack --mode production",
+    "build": "webpack --mode production && npm run cp",
```

Comment:
Same as above.

Reply:
These come from scaffolding tools office provided. Modifying and serving add-in manifest file relies heavily on webpack plugins, and can be tricky when migrating. As add-in is a relatively separate project, I think we can leave as it is.